lib/matplotlib/text.py

```diff
@@ -195,6 +196,7 @@ def __init__(self,
                  rotation=None,
                  linespacing=None,
                  rotation_mode=None,
+                 trans_rotate=False,
```

Comment:
Please put this last to maintain backcompatibility of the API.  The issue is users are (unwisely!) passing all of this as positional args.